README.md

```diff
@@ -3,8 +3,37 @@
 
 # dwave-scikit-learn-plugin
 
+This package contains a `sci-kit learn` transformer which does hybrid feature selection. For more information about the hybrid feature selection method, see the [example notebook](https://github.com/dwave-examples/feature-selection-notebook).
```

Comment:
```suggestion
This package provides a `scikit-learn` transformer for 
[feature selection](https://en.wikipedia.org/wiki/Feature_selection) using a quantum-classical hybrid solver. 

For an introduction to hybrid methods for feature selection, see the [Feature Selection example](https://github.com/dwave-examples/feature-selection-notebook) Jupyter Notebook.
```

* [scikit](https://scikit-learn.org/stable/) write it "scikit-learn"
* Try to keep lines below 100 chars
* I don't think we should introduce a new "hybrid feature selection" terminology, we're applying a quantum-classical hybrid solution method to the classical problem of feature selection